Sources/DistributedCluster/ClusterSystem.swift

```diff
@@ -1049,6 +1049,14 @@ extension ClusterSystem {
         if let wellKnownName = actor.id.metadata.wellKnown {
             self._managedWellKnownDistributedActors[wellKnownName] = actor
         }
+      
+        if let eventSourced = actor as? (any EventSourced) {
+            Task { [weak eventSourced] in
+              await eventSourced?.whenLocal { [weak self] myself in
+                await self?.journal.restoreEventsFor(actor: myself, id: myself.persistenceId)
+              }
+            }
```

Comment:
As journal is an actor—not sure how to escape `Task` here...
On the other hand feels like ok, as the idea is just to send command to actor to restore state.
What should happen though if recovery fails? Guess there should be some recovery failed event from journal plugin?

Reply:
Tbh I’d make journal an actor maybe? Why not? Just a local actor I mean :)

Reply:
I mean, it's already an actor, so you need an `await` then